source/css/style.css

```diff
@@ -243,6 +243,11 @@ h1, h2, h3, h4, h5, h6 {
 .paddings{
 	padding: 70px 0;
 }
+@media (max-width: 991px) {

```

Comment:
I guess the problem was that we incorrectly started editing the `css` file at some point in the past, so now it's hard to incorporate the changes backwards. We should probably clean up the whole thing at some point.